Sources/ApodiniExtension/AsyncSequenceHelpers/AsyncMergeSequence.swift

```diff
@@ -0,0 +1,232 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import Foundation
+import _Concurrency
+
+public struct AsyncMergeSequence<Base, Other>: AsyncSequence where Base: AsyncSequence, Other: AsyncSequence, Base.Element == Other.Element {
```

Comment:
Maybe some short docs defining this particular AsyncSequence. Maybe including some words about what promises are made or not made in terms of sequence, as far as I see there are none?

Reply:
Yeah, there should be at least a short description. I put the real discussion on the `AsyncSequence` extension (`.merge(with:)`). That's also how it is done in the standard libs' docs.